ssm.py

```diff
@@ -0,0 +1,11 @@
+import boto3
+
+
+def getPasswords():
+  ssm = boto3.client('ssm')
+  passwrd = ssm.get_parameter(Name='/prod/nyp/snowflake-db/stack-user-pasword', WithDecryption=True)
+  nyp_superset = passwrd['Parameter']['Value']
+  return nyp_superset
+
+def ssm_password_lookup(url):
```

Comment:
Use the url object passed in to fetch the RDS or snowflake password as appropriate

Reply:
I tested it, it does not seem to customize the passwords based on URLs